test/fixtures/wpt/FileAPI/BlobURL/test2-manual.html

```diff
@@ -51,7 +51,7 @@
 
                 xhr.send();
 
-            }, 'Check whether the browser response 500 in XHR if the selected file which File/Blob URL refered is not found');
+            }, 'Check whether the browser response 500 in XHR if the selected file which File/Blob URL referred is not found');
```

Comment:
This won't work. Everything inside `test/fixtures/wpt` is pulled from [web-platform-tests/wpt](https://github.com/web-platform-tests/wpt), and we frequently pull in upstream changes using `git node wpt` [(see docs)](https://github.com/nodejs/node/blob/v22.9.0/test/wpt/README.md#how-to-update-tests-for-a-module). Any changes to these files should be committed on the upstream repository instead, and then pulled back into Node afterwards.

I see that #55063 also touched these files. I highly recommend we revert those changes, since they'll be lost whenever we run `git node wpt` again.

Reply:
Done. All `wpt` fixes reverted including the ones in my other merged PR.

Reply:
BTW, I did consider giving `wpt` the full treatment upstream but it's absolutely riddled with typos so I shelved it for now.